reconcile/test/test_utils_jinja2.py

```
@@ -124,6 +124,12 @@ def test_json_pointers() -> None:
     assert json_pointers(input, "items[?@.name=='c']") == []
 
 
+def test_json_pointers_escapes_rfc6901_special_characters() -> None:
+    input = {"a/b": "slash", "a~b": "tilde"}
```

Ruff (0.16.0)

[error] Variable `input` is shadowing a Python builtin

(A001)